doc/mf6io/framework/array_data.tex

```diff
@@ -121,41 +121,21 @@ \subsubsection{READARRAY Examples}
-
-The following is simple example of a PERIOD block for the GHB Package, which shows how a list is entered by the user.
+In the next example, the ``LAYERED'' keyword is present and binary files are used for each layer. A control line with the ``BINARY'' keyword is required for each layer.
 
 \begin{lstlisting}[style=inputfile]
-BEGIN PERIOD 1
-#      lay       row       col     stage      cond
-         1        13         1     988.0     0.038
-         1        14         9    1045.0     0.038
-END PERIOD
-\end{lstlisting}
-
-As described earlier in the section on ``Block and Keyword Input,'' block information can be read from a separate text file.  To activate reading a list from separate text file, the first and only entry in the block must be a control line of the following form:  
-
-\begin{lstlisting}[style=blockdefinition]
-  OPEN/CLOSE <fname>
+  STRT LAYERED
+     OPEN/CLOSE strt.layer1.bin (BINARY)  #layer1
+     OPEN/CLOSE strt.layer2.bin (BINARY)  #layer2
+     OPEN/CLOSE strt.layer3.bin (BINARY)  #layer3
+     OPEN/CLOSE strt.layer4.bin (BINARY)  #layer4
 \end{lstlisting}
 
-\noindent where \texttt{fname} is the name of the file containing the list.  Lists for the stress packages (CHD, WEL, DRN, RIV, GHB, RCH, and EVT) have an additional BINARY option.  The BINARY  option is not supported for the advanced stress packages (LAK, MAW, SFR, UZF).  The BINARY options is specified as follows:
+In the next example, the ``LAYERED'' keyword is absent. In this case, a single control line with the ``BINARY'' keyword is required and the binary file will include the entire data array.
 
-\begin{lstlisting}[style=blockdefinition]
-  OPEN/CLOSE <fname> [(BINARY)]
+\begin{lstlisting}[style=inputfile]
+  STRT
+     OPEN/CLOSE strt.bin (BINARY)  #layers1-4
 \end{lstlisting}
 
-If the (BINARY) keyword is found on the control line, then the file is opened as an unformatted file on unit 99, and the list is read.  There are a number of requirements for using the (BINARY) option for lists.  All stress package lists begin with integer values for the \texttt{cellid} (layer, row, and column, for example).  These values must be represented as integer numbers in the unformatted file.  Also, all auxiliary data must be included in the binary file; auxiliary data must be represented as double precision numbers.  Lastly, the (BINARY) option does not support entry of \texttt{boundname}, and so the BOUNDNAMES option should not be activated in the OPTIONS block for the package.  
\ No newline at end of file
+A consequence of the way binary input files have been implemented in \mf, simulated dependent variable binary output (for example, head and concentration) cannot be used as binary array input for a model. Instead, simulated dependent variable binary output must be processed and split into separate binary files for each layer or combined into a single three-dimensional array.
```

Comment:
```suggestion
A consequence of the way binary input files have been implemented in \mf, simulated dependent variable binary output (for example, head and concentration) cannot be used as binary array input for a model. Instead, simulated dependent variable binary output must be processed and split into separate binary files for each layer or combined into a single array equal to the size of the grid (for DIS grids this would be an array equal to NCOL * NROW * NLAY).
```